tests/kafkatest/services/verifiable_producer.py

```diff
@@ -23,16 +23,18 @@ class VerifiableProducer(BackgroundThreadService):
     logs = {
         "producer_log": {
             "path": "/mnt/producer.log",
-            "collect_default": False}
+            "collect_default": True}
```

Comment:
Why did this change? Isn't there a data size problem since it logs every single message?